docs/_articles/en/einstein/einstein-example-prompts.md

```diff
@@ -5,11 +5,11 @@ lang: en
 
 ## Overview
 
-A prompt is natural language text that you craft to communicate with an AI. The response that’s generated depends on the quality of the prompt. Use prompt engineering to define the prompt and then refine it until you get a response that works for you. For E4D responses, prompt quality determines the quality of the Apex code that’s generated.
+A prompt is natural language text that you craft to communicate with an AI. The response that’s generated depends on the quality of the prompt. Use prompt engineering to define the prompt and then refine it until you get a response that works for you. For Einstein for Developer responses, prompt quality determines the quality of the Apex code that’s generated.
 
 ## About Generated Code Quality
 
-As with all LLM-based products, the technology behind Einstein for Developers is still new. The generated output often requires editing to be customized for your individual project. Some responses aren't optimal. We'd love to hear feedback from you on the generated output through the History & Feedback Panel in the IDE. Your feedback is critical to the continuous training of our large language model (LLM) during the beta program.
+As with all LLM-based products, the technology behind Einstein for Developers is still new. The generated output often requires editing to be customized for your individual project. Some responses aren't optimal.
```

Comment:
Can we ask them to share feedback on generated responses via our Github Issues? We're kinda hinting at it in the next paragraph